content/en/tracing/api_catalog/get_started.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Setting Up API Catalog
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+{{< site-region region="gov,ap1" >}}
+<div class="alert alert-warning">API Catalog is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+API Catalog uses APM instrumentation for distributed tracing to discover and list endpoints in all environments in your Datadog organization. Endpoints for instrumented services and supported libraries are automatically populated into API Explorer.
+
+## Setting up
+
+Because it is built on APM data, you can turn on API Catalog in a Datadog organization that is already consuming APM data, and no other setup required. APM instrumented services and supported libraries are automatically discoverable. 
+
+For APIs that are not yet instrumented, you can do one of the following to add it to the API Catalog:
```

Comment:
I think we should split it to two use cases:
1. for services that are not instrumented -> instruction to instrument the service
2. for services that are not supported -> import an openAPI file

I it is to much i would just change the text to: "for services that are not supported" instead of "for APIs that are not instrumented"


Another way to look at this could be something like this:
If you cannot find your API endpoints -> check that the underlying service is instrumented -> if it is instrumented but not supported, uploading an openAPI file would help finding those API endpoints